<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>871</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>–í—Ö–æ–¥–Ω–æ–π —Ñ–∞–π–ª:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputFileEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton">
         <property name="text">
          <string>–û–±–∑–æ—Ä...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>135</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>–ö–æ–¥–µ–∫:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="codecCombo">
       <item>
        <property name="text">
         <string>libx264</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>libx265</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>copy</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>–ö–æ–Ω—Ç–µ–π–Ω–µ—Ä:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="containerCombo">
       <item>
        <property name="text">
         <string>mp4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mkv</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>–†–∞–∑—Ä–µ—à–µ–Ω–∏–µ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="resolutionCombo">
       <item>
        <property name="text">
         <string>480p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>720p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1080p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>custom</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>340</y>
      <width>237</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Custom (width:height):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="customResolutionEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>1920:1080</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>–ö–æ–º–∞–Ω–¥–∞ FFmpeg:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>871</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QTextEdit" name="commandDisplay">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #f0f0f0; font-family: Consolas;</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyCmdButton">
       <property name="text">
        <string>–ö–æ–ø–∏—Ä–æ–≤–∞—Ç—å</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>871</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="styleSheet">
        <string notr="true">background-color: #4CAF50; color: white; font-weight: bold; padding: 8px;</string>
       </property>
       <property name="text">
        <string>–ó–∞–ø—É—Å—Ç–∏—Ç—å –∫–æ–¥–∏—Ä–æ–≤–∞–Ω–∏–µ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseResumeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>‚è∏ –ü–∞—É–∑–∞</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savePresetButton">
       <property name="text">
        <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å –ø—Ä–µ—Å–µ—Ç</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadPresetButton">
       <property name="text">
        <string>–ó–∞–≥—Ä—É–∑–∏—Ç—å –ø—Ä–µ—Å–µ—Ç</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletePresetButton">
       <property name="text">
        <string>–£–¥–∞–ª–∏—Ç—å –ø—Ä–µ—Å–µ—Ç</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportPresetButton">
       <property name="text">
        <string>–≠–∫—Å–ø–æ—Ä—Ç –ø—Ä–µ—Å–µ—Ç–æ–≤</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importPresetButton">
       <property name="text">
        <string>–ò–º–ø–æ—Ä—Ç –ø—Ä–µ—Å–µ—Ç–æ–≤</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>–õ–æ–≥ –≤—ã–ø–æ–ª–Ω–µ–Ω–∏—è:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="logDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>871</width>
      <height>101</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="openOutputFolderButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>–û—Ç–∫—Ä—ã—Ç—å –ø–∞–ø–∫—É</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-40</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="videoPreviewContainer" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="6,2,1">
      <item>
       <widget class="QWidget" name="videoPreviewWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #cccccc; background-color: #000000;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="videoTimelineSlider">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="videoControlsLayout" stretch="2,2,0,0">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QPushButton" name="videoPlayButton">
          <property name="text">
           <string>‚ñ∂ Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="videoStopButton">
          <property name="text">
           <string>‚èπ Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="videoTimeLabel">
          <property name="text">
           <string>00:00 / 00:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="videoMuteButton">
          <property name="text">
           <string>‚Äãüîä</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QProgressBar" name="encodingProgressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>871</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumainwindow">
    <property name="title">
     <string>mainwindow</string>
    </property>
   </widget>
   <addaction name="menumainwindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
